coral/src/domain/acl/acl-api.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  CreateAclRequestTopicTypeProducer,
+  CreateAclRequestTopicTypeConsumer,
+} from "src/domain/acl/acl-types";
+import api from "src/services/api";
+import { KlawApiResponse } from "types/utils";
+
+const postAclRequest = (
+  aclParams:
+    | CreateAclRequestTopicTypeProducer
+    | CreateAclRequestTopicTypeConsumer
+): Promise<KlawApiResponse<"createAclRequest">> => {
+  return api.post("/createAcl", aclParams);
```

Comment:
We need to define the `response` and `request` types for the API function. It is quite verbose, but without the types, one could try to pass any data to the function without type error.
```
api.post<KlawApiResponse<"createAclRequest">, KlawApiRequest<"createAclRequest">>("/createAcl", aclParams);
```

Reply:
Ah, very nice, sorry for the oversight!